src/private/conversations.ts

```diff
@@ -0,0 +1,66 @@
+import { ensureInitialized, sendMessageRequest } from "../internal/internalAPIs";
+import { GlobalVars } from "../internal/globalVars";
+import { frameContexts } from "../internal/constants";
+import { StartConversationRequest, ShowConversationRequest } from "../public/interfaces";
+
+/**
+ * Namespace to interact with the conversational subEntities inside the tab
+ */
+export namespace conversations {
+
+  /**
+  * @private
+  * Hide from docs
+  * --------------
+  * Allows the user to start a conversation with each subentity inside a tab
+  * @param startConversationRequest Callback containing the conversation Id and if the tab pane was closed
+  */
+  export function startConversation(
+    startConversationRequest: StartConversationRequest
+  ): void {
+    ensureInitialized(frameContexts.content);
+    const messageId = sendMessageRequest(GlobalVars.parentWindow, "conversations.startConversation", [{
+      title: startConversationRequest.title,
+      subEntityId: startConversationRequest.subEntityId
+    }]);
+    GlobalVars.callbacks[messageId] = (conversationId?: string, reason?: string) => {
```

Comment:
This callback only gets invoked a single time when the handler for this message on the Teams side completes; however, my expectation is that we might have to call both `onStartConversation` when the user posts the first message and then again `onCloseConversation` some time later when the user dismisses the chat pane. How do you plan to trigger both of these handlers for a single call to startConversation?